addons/account/models/account_invoice.py

```diff
@@ -1346,6 +1396,27 @@ def _set_additional_fields(self, invoice):
         """
         pass
 
+    edi_description = fields.Char(string='EDI description', compute='_compute_edi_description')
+    edi_product_name = fields.Char(string='EDI product name', compute='_compute_edi_product_name')
```

Comment:
Can you extend this fields with a little help, I do nopt understand the meaning of them easily